File: app/src/test/java/com/example/jonathan/webjsonviewer/FriendTest.java
package com.example.jonathan.webjsonviewer;

import static org.junit.Assert.assertEquals;

import com.example.jonathan.webjsonviewer.model.Friend;

import org.junit.Test;

public class FriendTest {
  @Test
  public void testAccessId() {
    int id = 12345;
    Friend aFriend = new Friend();
    aFriend.setId(id);
    assertEquals(id, aFriend.getId());
  }

  @Test
  public void testAccessName() {
    String name = "John Smith";
    Friend aFriend = new Friend();
    aFriend.setName(name);
    assertEquals(name, aFriend.getName());
  }
}

File: app/src/test/java/com/example/jonathan/retrofitwebjsonviewer/FriendTest.kt
package com.example.jonathan.retrofitwebjsonviewer

//JZ import com.example.jonathan.retrofitwebjsonviewer.model.Friend.id
//JZ import com.example.jonathan.retrofitwebjsonviewer.model.Friend.name
import com.example.jonathan.retrofitwebjsonviewer.model.Friend
import org.junit.Assert
import org.junit.Test

class FriendTest {
    @Test
    fun testAccessId() {
        val id = 12345
        val aFriend = Friend()
        aFriend.id = id
        Assert.assertEquals(id.toLong(), aFriend.id.toLong())
    }

    @Test
    fun testAccessName() {
        val name = "John Smith"
        val aFriend = Friend()
        aFriend.name = name
        Assert.assertEquals(name, aFriend.name)
    }
}